web/src/components/tasks/business-formation/review/ReviewStep.test.tsx

```diff
@@ -773,4 +773,28 @@ describe("Formation - ReviewStep", () => {
       });
```

Comment:
[dust] This is just a general comment, not related to your work but I'm looking through this test file and it's pretty difficult to read because there isn't a structured grouping of any sort? I think we should open a refactor ticket to organize it by section like below or whichever other grouping makes sense because it's disorganized and will grow to be more chaotic in the future? 

```
describe("business"){
    describe("llp")
    describe("llc")
    ....
}

describe("Contacts"){
    describe("llp")
    describe("llc")
    ....
}
```